test/solution.jl

```diff
@@ -134,7 +134,7 @@ using ODEProblemLibrary: prob_ode_lotkavolterra
                 @test_nowarn plot(sol, denseplot=false)
                 message = "This plot does not visualize any uncertainties"
                 @test_logs (:warn, message) plot(sol, idxs=(1, 2))
-                @test_logs (:warn, message) plot(sol, idxs=(1, 1, 2))
```

Comment:
This is broken test is precisely what relates to the comment above. There are probably worse cases though, e.g. with idxs being the correct length but unexpected order, so this should definitely get fixed!